apps/coordinator/src/components/ScriptExplorer/Transaction.jsx

```diff
@@ -2,8 +2,9 @@ import React from "react";
 import PropTypes from "prop-types";
 import { connect } from "react-redux";
 import {
-  signedMultisigTransaction,
```

Comment:
This is a second function that's replaced (it uses `unsignedMultisigTransaction`, one of the others that needed to be upgraded)